src/functions/template-tags/organizer.php

```diff
@@ -253,34 +253,64 @@ function tribe_get_organizer_email( $postId = null, $antispambot = true ) {
 	 *
 	 * @param int  $postId    Can supply either event id or organizer id, if none specified, current post is used
 	 * @param bool $full_link If true outputs a complete HTML <a> link, otherwise only the URL is output
-	 * @param bool $echo      If true, echo the link, otherwise return
+	 * @param bool $echo      Deprecated. If true, echo the link, otherwise return
 	 *
 	 * @return string Organizer Name and Url
 	 */
 	function tribe_get_organizer_link( $postId = null, $full_link = true, $echo = false ) {
 
 		// As of TEC 4.0 this argument is deprecated
 		// If needed precede the call to this function with echo
-		if ( $echo != false ) _deprecated_argument( __FUNCTION__, '4.0' );
+		if ( $echo != false ) _deprecated_argument( __FUNCTION__, '4.0', 'As of TEC 4.0 this argument is deprecated. If needed, precede the call to this function with echo' );
```

Comment:
* :no_entry_sign: Inline control structures are not allowed
* :no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 46cb13082d5a3882888b9037d862af1f</sub>